[2023-09-04T06:08:15.870+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-11-20T00:00:00+00:00 [queued]>
[2023-09-04T06:08:15.882+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-11-20T00:00:00+00:00 [queued]>
[2023-09-04T06:08:15.883+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-04T06:08:15.900+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2022-11-20 00:00:00+00:00
[2023-09-04T06:08:15.908+0000] {standard_task_runner.py:57} INFO - Started process 8249 to run task
[2023-09-04T06:08:15.911+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v04', 'insert_into_table', 'scheduled__2022-11-20T00:00:00+00:00', '--job-id', '1682', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmpgo1ne8u0']
[2023-09-04T06:08:15.915+0000] {standard_task_runner.py:85} INFO - Job 1682: Subtask insert_into_table
[2023-09-04T06:08:16.036+0000] {task_command.py:415} INFO - Running <TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-11-20T00:00:00+00:00 [running]> on host 10bdc2e98230
[2023-09-04T06:08:16.208+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mohghaff' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v04' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2022-11-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-11-20T00:00:00+00:00'
[2023-09-04T06:08:16.209+0000] {sql.py:274} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2022-11-20', 'dag_with_postgres_operator_v04')
        
[2023-09-04T06:08:16.221+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:08:16.231+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:08:16.247+0000] {sql.py:418} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2022-11-20', 'dag_with_postgres_operator_v04')
        , parameters: None
[2023-09-04T06:08:16.252+0000] {sql.py:427} INFO - Rows affected: 1
[2023-09-04T06:08:16.273+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v04, task_id=insert_into_table, execution_date=20221120T000000, start_date=20230904T060815, end_date=20230904T060816
[2023-09-04T06:08:16.325+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-04T06:08:16.342+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
